scaletest/templates/scaletest-runner/scripts/run.sh

```diff
@@ -6,54 +6,61 @@ set -euo pipefail
 # shellcheck disable=SC2153 source=scaletest/templates/scaletest-runner/scripts/lib.sh
 . "${SCRIPTS_DIR}/lib.sh"
 
+mapfile -t scaletest_load_scenarios < <(jq -r '. | join ("\n")' <<<"${SCALETEST_PARAM_LOAD_SCENARIOS}")
+export SCALETEST_PARAM_LOAD_SCENARIOS=("${scaletest_load_scenarios[@]}")
+
 log "Running scaletest..."
 set_status Running
 
 start_phase "Creating workspaces"
 coder exp scaletest create-workspaces \
-	--count "${SCALETEST_NUM_WORKSPACES}" \
-	--template "${SCALETEST_TEMPLATE}" \
-	--concurrency "${SCALETEST_CREATE_CONCURRENCY}" \
-	--job-timeout 15m \
+	--count "${SCALETEST_PARAM_NUM_WORKSPACES}" \
+	--template "${SCALETEST_PARAM_TEMPLATE}" \
+	--concurrency "${SCALETEST_PARAM_CREATE_CONCURRENCY}" \
+	--job-timeout 2h \
 	--no-cleanup \
 	--output json:"${SCALETEST_RESULTS_DIR}/create-workspaces.json"
```

Comment:
nit: Since you're prefixing everything else with `SCALETEST_PARAM_`, perhaps also this one?

Reply:
I was thinking about them separately, one is params that a user can change (via template) and the other is plain variables used by the suite. Since all `SCALETEST_PARAM_` prefixed vars are attached to reports in sorted order, this also gives us more control what to report/where.